requirements.txt

```diff
@@ -4,28 +4,28 @@ albumentations>=1.3.0
 Click>=8.1.3
 defusedxml>=0.7.1
 flask>=2.2.2
-glymur>=0.11.7
-imagecodecs>=2021.11.20
-jupyterlab>=3.4.8
-matplotlib>=3.5.3
-numpy>=1.21.6
+glymur>=0.12.1
+imagecodecs>=2022.9.26
+jupyterlab>=3.5.2
+matplotlib>=3.6.2
+numpy>=1.23.5
 opencv-python>=4.6.0
 openslide-python>=1.2.0
-pandas>=1.3.5
-pillow>=9.2.0
-pydicom>=2.3.0 # Used by wsidicom
+pandas>=1.5.2
+pillow>=9.3.0
+pydicom>=2.3.1 # Used by wsidicom
 pyyaml>=6.0
 requests>=2.28.1
 scikit-image>=0.19.3
-scikit-learn>=1.0.2
-scipy>=1.7.3
+scikit-learn>=1.2.0
+scipy>=1.9.3
 shapely>=1.8.5, <2.0.0 # Remove 2.0.0 once AnnotationStore is compatible
-SimpleITK>=2.1.1
-sphinx>=4.3.2
-tifffile>=2021.11.2
-torch>=1.13.0
-torchvision>=0.13.0
+SimpleITK>=2.2.1
+sphinx>=5.3.0
+tifffile>=2022.10.10
+torch>=1.13.1
+torchvision>=0.14.1
 tqdm>=4.64.1
 umap-learn>=0.5.3
-wsidicom>=0.3.2
-zarr>=2.12.0
+wsidicom>=0.5.0
+zarr>=2.13.3
```

Comment:
Changing pinned versions should probably be a separate PR as unrelated to dropping 3.7

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/526/commits/3e370499a7edc12328fb86034c0dbe8be725b9d4